spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -1605,7 +1605,43 @@ def over_65_birth_year
       end
     end
 
-    context 'when no w2s' do
+    context 'when has only 1099-Rs' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_minimal)}
+      let!(:state_file_1099r) { create :state_file1099_r, intake: intake, state_tax_withheld_amount: 100.1 }
```

Comment:
[pebble] might be nice to have multiple 1099Rs here to check that it's summing something

Reply:
done - i also combined this with the discussion about `income_source` so it sums from all different sources to make clear it works for all